app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts_of_command = command.split()
+    file_source = parts_of_command[1]
+    file_new = parts_of_command[2]
+    condition = [
+        command[:2] == "cp",
```

Comment:
Use unpacking after checking the length of the element after split to avoid indexing, make code more descriptive and reducing the number of conditions to check